src/pipecat/frames/frames.py

```diff
@@ -404,6 +434,52 @@ class BotInterruptionFrame(SystemFrame):
     pass
 
 
+@dataclass
+class BotStartedSpeakingFrame(SystemFrame):
+    """Emitted upstream by transport outputs to indicate the bot started speaking."""
+
+    pass
+
+
+@dataclass
+class BotStoppedSpeakingFrame(SystemFrame):
+    """Emitted upstream by transport outputs to indicate the bot stopped speaking."""
+
+    pass
+
+
+@dataclass
+class BotSpeakingFrame(SystemFrame):
```

Comment:
All these three are now SystemFrames for the same reasons I mentioned. We just want to know about it quickly.